agent360/agent360.py

```diff
@@ -327,7 +327,7 @@ def _config_init(self):
             'subprocess': 'no',
             'user': '',
             'server': '',
-            'api_host': 'ingest.monitoring360.io',
+            'api_host': 'api.monitoring360.io',
```

Comment:
We probably want to have 2 different hosts for both the hello command as well as the ingestion of server data payloads.

Reply:
Updated